.luacheckrc

```diff
@@ -0,0 +1,28 @@
+unused_args = false
+allow_defined_top = true
+max_line_length = 999
+
+ignore = {
+    "pos1", "pos2",
+    "extent1", "extent2",
+    "header", "count",
+
+    "err", "state",
+    "filename", "def",
+    "file", "name",
+}
```

Comment:
why are these ignored?

Reply:
Test locally and you'll see what happens.